### DISCUSSION TITLE
How do I hide input parameters in Python?

### DISCUSSION THREAD

**Discussion by uriahf** [quarto-dev/quarto-cli#1465] [Q&A]
Hi all! 

I want to render qmd file with Jupyter engine, but I'm not sure how to hide the input parameters. I tried to use echo: false but it does not work. Any ideas? 

Thanks!

Reference:
https://quarto.org/docs/computations/parameters.html

**Comment by rgaiacs** [comment]
Hi @uriahf. Can you provide a minimal working example? `echo: false` works for me.

**Reply by uriahf** [reply]
It appears that `echo: false` works for me when I use the vscode button, but not when I use the Terminal.

### YAML  in reprex.qmd:

```
---
title: "Untitled"
format: docx
---
```


### Chunk for Parameters in reprex.qmd:

```{python}
#| echo: false
#| tags: [parameters]

name = "uriah"
```
### Chunk for output in reprex.qmd:

```{python}
#| echo: false
#| output: asis

print(name + " knows how to use python")
```

### Python code in a separate file:

```
import os
os.system("quarto render reprex.qmd  -P name:someone")
```
